modules/ROOT/pages/contributing/source-git.adoc

```diff
@@ -4,55 +4,69 @@ include::partial$attributes.adoc[]
 
-== Submitting a change in a source-git repo
+=== Submitting a change in a source-git repo
 
-If you want to update packaging files (in SPECS directory), you can go ahead and change them - no special care is needed.
+Changes to packaging files, such as the spec file, are done without any special care on your side: just edit the file, save it and commit it. All the packaging files live in a `.distro` subidrectory.
 
 === Changing source code by defining the patch in a spec file
 
-Creating source code changes needs a bit more work. Packages are modified in CentOS Stream 8 traditional dist-git workflow by creating patch files and defining those patches in a spec file. The patches are then applied during a build process in a `%prep` section.
+Creating source code changes needs a bit more work. Packages are modified in CentOS Stream 9 traditional dist-git workflow by creating patch files and defining those patches in a spec file. The patches are then applied during a build process in a `%prep` section.
 
 If you are changing source code and you know that a patch in the spec is needed, add the patch to the spec file with a name of your choice:
+
 ----
 Patch123: my-special-change.patch
 ----
-Now you need to annotate commit message with the respective change
+
+Now you need to annotate the git commit which matches the patch with the respective change
 ----
 We need this change in CentOS Stream
 
 patch_name: my-special-change.patch
-present_in_specfile: true
 
 Signed-off-by: Tomas Tomecek <ttomecek@redhat.com>
+
 ---
  src/code.c | 17 +++++++++--------
  1 file changed, 9 insertions(+), 8 deletions(-)
 ----
-The automation, packit, then reads the commit message, knows that the patch is already defined in the spec and creates a patch file with the name set in `patch_name` from the respective commit when creating a SRPM.
 
-This is the preferred way of changing source code since you are in full control of the process.
+The automation, packit, then reads the commit message, knows that the patch is already defined in the spec file and creates a patch file with the name set in `patch_name` from the respective commit when creating a SRPM.
+
+This is the preferred way of changing source code since you define the name of patch files.
+
+=== After a merge request is created
 
-=== Changing source code: packit takes care of patches
+You should receive a review from a Red Hat maintainer.
 
-If packit does not find such metadata as defined above in a commit message:
+Once your MR is approved, it's gonna be merged first in a respective dist-git repo to make sure it can built and passes all the checks. Once all is good, it can be merged in the source-git repo as well.
 
-1. it adds a new patch entry in the spec file
-2. and generates a patch file from the respective commit
+When you open a source-git MR, you'll be able to see CI checks which are comming from a mirrored MR which was autoamtically created in dist-git.
```

Comment:
```suggestion
When you open a source-git MR, you'll be able to see CI checks which are coming from a mirrored MR which was automatically created in dist-git.
```

Reply:
uhhh, I seem not to have spellcheck set up on my new workstation -_-

Reply:
okay, so it's .adoc files, I have spellcheck enabled for markdown, not other text files